frontend/src/components/recipe_card/recipe_card.js

```diff
@@ -4,24 +4,50 @@ import './recipe_card.css';
 
```

Comment:
This may seem mildly insane, but can we change all these `recipe_card` file names to use camel case, like `RecipeCard`? It's pretty standard for components to use that format.

Reply:
Done!